src/agent/scm/lib/tfvcwrapper.ts

```diff
@@ -134,27 +151,11 @@ export class TfvcWrapper extends events.EventEmitter {
 
         return msg;
     }
-
-    private _exec(cmd: string, args: string[], options?: ITfvcExecOptions): Q.Promise<number> {
-        if (this.tfPath === null) {
-            return this._getTfNotInstalled();
-        }
-
+    
+    private _getToolRunner(cmd: string, args: string[]) {
         var tf = new tl.ToolRunner(this.tfPath);
```

Comment:
I am working on this now.  Want to make sure you are ok with how I moved the resolveInputPath() after getCode() first.